lib/tapioca/dsl/compilers/active_record_columns.rb

```diff
@@ -150,13 +176,25 @@ def gather_constants
         sig { returns(Helpers::ActiveRecordColumnTypeHelper::ColumnTypeOption) }
         def column_type_option
           @column_type_option ||= T.let(
-            Helpers::ActiveRecordColumnTypeHelper::ColumnTypeOption.from_serialized(
-              options.fetch(
-                "types",
-                Helpers::ActiveRecordColumnTypeHelper::ColumnTypeOption::Persisted.serialize,
-              ),
-            ),
-            T.nilable(Helpers::ActiveRecordColumnTypeHelper::ColumnTypeOption),
+            begin
+              column_type_option = Helpers::ActiveRecordColumnTypeHelper::ColumnTypeOption::Persisted
+
+              if options.key?("ActiveRecordColumnTypes")
+                value = options["ActiveRecordColumnTypes"]
```

Comment:
```suggestion
              value = options["ActiveRecordColumnTypes"]
              if value
```

Reply:
We could even do something like `options["ActiveRecordColumnTypes"] || "persisted"` but this forces using the string rather than the constant for `"persisted"`.

Reply:
I wonder if we could move this logic in the enum itself: `ColumnTypeOption.from_option`?